[3.3.5-proc] Unit::HandleProcTriggerSpell

ccrs opened the pull request:
@ariel-

crsdev committed:
Improved Drain Soul

crsdev committed:
travis

crsdev committed:
Deep Wounds Aura

crsdev committed:
Persistent Shield

I know its not blizzlike, but its better! :D

crsdev committed:
missing sanity checks around GetHealInfo()

airel dont kill me for including this, but think about those empty heals and crashes, think about em t.t

crsdev committed:
Unyielding Knights

crsdev committed:
some corrections

crsdev committed:
Deflection

crsdev committed:
some corrections

Exodius reviewed (changes requested):
I don't want to sound bitchy but, shouldn't these be separated into separate PRs?

ccrs commented:
I'm helping ariel- with the proc system revamp, dw

crsdev committed:
Cheat Death

crsdev committed:
some corrections

crsdev committed:
Greater Heal Refund

crsdev committed:
Bonus Healing (Crystal Spire of Karabor)

crsdev committed:
Decimation

crsdev committed:
Blood Beast's Blood Link

crsdev committed:
Improved Spirit Tap

crsdev committed:
some corrections

if someone can tell me a better solution for the spellmgr change, I'm open to suggestions

crsdev committed:
Ooze Spell Tank Protection

crsdev committed:
hp % on damage  trinkets

crsdev committed:
small correction

crsdev committed:
Blade Barrier

crsdev committed:
Rime

crsdev committed:
rename

crsdev committed:
Turn the Tables

crsdev committed:
small correction

crsdev committed:
Setup

crsdev committed:
Improved Lay on Hands

crsdev committed:
small correction

crsdev committed:
Finisher Combo & Ruthlessness

Keader commented:
This branch has conflicts that must be resolved
Only those with write access to this repository can merge pull requests.
Conflicting files
src/server/scripts/Spells/spell_warrior.cpp

crsdev committed:
Omen of Clarity

crsdev committed:
small correction

crsdev committed:
Shamanistic Rage

crsdev committed:
some corrections

crsdev committed:
Enlightenment

crsdev committed:
Demonic Power

crsdev committed:
small correction

crsdev committed:
Warrior extra proc

crsdev committed:
Sword and Board

crsdev committed:
correction

crsdev committed:
Burning Determination & Astral Shift

crsdev committed:
Burning Determination proc's proc

ariel- committed:
Scarab brooch does not replace stronger shields

ariel- committed:
Typo fix (bellow -> below), remove unused parameters
